game/scripts/vscripts/abilities/twin/modifier_boss_twin_twin_empathy.lua

```diff
@@ -0,0 +1,36 @@
+--LinkLuaModifier("modifier_boss_twin_twin_empathy", "abilities/twin/boss_twin_twin_empathy.lua", LUA_MODIFIER_MOTION_NONE)
+
+modifier_boss_twin_twin_empathy_buff = class({})
+
+--This may need to be in the abil not the mod
+function modifier_boss_twin_twin_empathy_buff:OnCreate()
+  local interval = self:GetSpecialValueFor( "heal_timer" )
+  mod:StartIntervalThink(interval)
```

Comment:
Modifier intervals keep running until you run StartIntervalThnk(-1)... Either way `mod` isn't defined anywhere so you're not going to get any intervals until you change that to `self`.